libs/@local/hash-graph-sdk/node/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@local/hash-graph-sdk",
+  "version": "0.0.0-private",
+  "private": true,
+  "description": "SDK for the HASH Graph API.",
+  "license": "AGPL-3.0",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./pkg/src/*.ts",
+      "node": "./pkg/src/*.ts",
+      "default": "./pkg/src/*.ts"
```

what do we need the `pkg` folder for?

---

That's a copy/paste error 🙂 
```suggestion
      "types": "./src/*.ts",
      "node": "./src/*.ts",
      "default": "./src/*.ts"
```